README.md

```diff
@@ -48,9 +49,8 @@ Fields:
 ## Known limitations
 
 There are two big limitations that prevent this plugin from being useful for production:
-
-1. Until RethinkDB supports `return_initial` on all changefeeds ([which it should in 2.1](https://github.com/rethinkdb/rethinkdb/issues/3579)), this plugin won't do "backfilling" of documents already in a table. It only sends changes to the table as they are received. When `return_initial` is implemented fully, this plugin will be updated to take advantage of that capability to provide backfilling.
-2. Until RethinkDB supports resuming changefeeds ([which it should in 2.1](https://github.com/rethinkdb/rethinkdb/issues/3471)), this plugin cannot guarantee that no changes are missed if a connection to the database is dropped. Again, once that functionality is implemented, this plugin will be modified to provide reliable "at least once" semantics for changes (meaning once it reconnects, it can catch back up and send any changes that it missed).
+1. Until RethinkDB supports resuming changefeeds ([which it should in 2.3](https://github.com/rethinkdb/rethinkdb/issues/3471)), this plugin cannot guarantee that no changes are missed if a connection to the database is dropped. Again, once that functionality is implemented, this plugin will be modified to provide reliable "at least once" semantics for changes (meaning once it reconnects, it can catch back up and send any changes that it missed).
+2. Since LogStash operates on a document by document basis, if the LogStash plugin gets disconnected from RethinkDB and a document is deleted, when the LogStash plugin reconnects it will not send a deletion event, even though it performs a backfill.
```

Comment:
This sentence is a bit complex. How about:

Documents that are deleted in RethinkDB while the LogStash plugin is disconnected will not be synchronized to LogStash. This is true even if `backfill` is enabled. This limitation is a consequence of LogStash operating on a document by document basis.